pkgs/tools/admin/exoscale-cli/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, buildGo114Package, fetchFromGitHub }:
+
+buildGo114Package {
+  pname = "exoscale-cli";
+  version = "1.12.0";
+
+  src = fetchFromGitHub {
+    owner  = "exoscale";
+    repo   = "cli";
+    rev    = "4b82b898b56ea238eb80ac38da40dc8a460a7de7";
+    sha256 = "04ym7mfv565icj3lmd2nrvq9asawwmmzg09681pj9py61ws56bxr";
+  };
+
+  goPackagePath = "github.com/exoscale/cli";
+  subPackages = [ "." ];
+  goDeps = ./deps.nix;
+
+  # can't use buildFlags here because these are passed to "go install"
+  # which does not recognize -o
```

Comment:
What's `-o`? If I didn't understand the comment right away, then perhaps it can be improved :).